src/index.html

```diff
@@ -1,14 +1,21 @@
-<!doctype html>
+<!Doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Snake</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Snake</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Snake</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="container">
+      <div class="block block--1">1</div>
```

Comment:
```suggestion
      <div class="block">1</div>
```
Можна використовувати nth-child замість того, щоб створювати безліч класів

Reply:
Виконано. Видалив класи block--N з кожного блоку в index.html, а також замінив стилі для класів ".block--N" на ".block:nth-child(N)" в файлі style.css.